source/controlTypes.py

```diff
@@ -427,7 +427,7 @@
 	# Translators: Identifies an alert message such as file download alert in Internet explorer 9 and above.
 	ROLE_ALERT:_("alert"),
 	# Translators: Identifies an internal frame (commonly called iframe; usually seen when browsing some sites with Internet Explorer).

```

Comment:
This obviously isn't your comment, but while we're here, let's clarify/simplify this a bit:

```
# Translators: Identifies an internal frame. This is usually a frame on a web page; i.e. a web page embedded within a web page.
```